### DISCUSSION TITLE
Delete beatmap directly in the editor

### DISCUSSION THREAD

**Discussion by nanashi-1** [ppy/osu#19997] [Q&A] [answered]
Being able to delete difficulty inside the editor sounds convenient.
What is stopping pull request (#19413) from being merge? Is it because it lacks Tests?

**Comment by peppy** [comment, marked as answer]
Since you found the PR, you should be able to see why it's blocked (yes, it requires tests and review).

If you are in a hurry, you can build yourself with the pull request applied. There's no need to open discussions for this.
